docs/getting-started-on-linux.md

```diff
@@ -0,0 +1,109 @@
+## Getting Started
+
+Follow the development machine setup on each desktop you'd like to use.
+
+### Development Machine Setup
+
+Clone the repository locally from terminal:
+
+```markdown
+> cd (to working directory of your choosing)
+> git clone https://github.com/microsoft/do-client
+```
+
+Run the appropriate bootstrapper depending on development machine platform:
+
+```markdown
+> cd build/bootstrap
+> ./bootstrap-ubuntu-18.04.sh
+```
+
+### Building DO client components 
+**NOTICE:**  
+**If you are modifying this project and distributing your own custom build, please modify the DO_BUILDER_IDENTIFIER cmake variable located in https://github.com/microsoft/do-client/blob/main/CMakeLists.txt**
+
```

Comment:
I think the link from the main README to this doc is sufficient. We should reduce the number of dependencies that ADU repo/docs have on ours.

---
In reply to: [605225054](https://github.com/microsoft/do-client/pull/17#discussion_r605225054) [](ancestors = 605225054)

Reply:
ADU public docs have specifically linked to a header in the main readme which now exists in this file. Once this is merged into main we will need to update the public docs. Actually @ValOlson  can you update the du public docs on du builder identifier to specifically point to the main README?

---
In reply to: [605245017](https://github.com/microsoft/do-client/pull/17#discussion_r605245017) [](ancestors = 605245017,605225054)

Reply:
I've decided to revert the header in the main README.md so that it doesn't break du docs linkage. It'll still point to this new doc where a user will see the instruction to modify builder name. I've synced with Val so she's aware of this change, but no action will be needed from the public docs.

---
In reply to: [605252365](https://github.com/microsoft/do-client/pull/17#discussion_r605252365) [](ancestors = 605252365,605245017,605225054)